platform/view/services/storage/driver/sql/postgres/test_utils.go

```diff
@@ -181,7 +182,28 @@ func startPostgresWithLogger(c ContainerConfig, t Logger, printLogs bool) (func(
 			Timeout:  time.Second,
 			Retries:  10,
 		},
-	}, &container.HostConfig{
+	}
+
+	// Add extended logging configuration if enabled
+	if c.ExtendedLogging {
+		containerConfig.Cmd = []string{
```

Comment:
The `ExtendedLogging` toggle probably makes sense for many cases and I can imagine that when debugging with postgres, sometimes more flexibility is needed when investigating an odd issue.

I think it would be nice to allow the `Cmd` be overwritten via env vars to let the user inject any postgres command / env vars via testing without touching this code all the time.